### PR TITLE
Add automated contrast checking

### PR DESCRIPTION
**Pull request by NickColley**
Seeing if we can get automated testing running in GitHub actions, we could add checks in other specs if this is viable.

As part of #513

### CODE REVIEW

File: .testcaferc.json [hunk 1]
```diff
@@ -5,5 +5,10 @@
     "takeOnFails": false
   },
   "skipJsErrors": true,
-  "stopOnFirstFail": true
-}
+  "stopOnFirstFail": true,
+  "clientScripts": [
+    {
+      "module": "axe-core/axe.min.js"
```
Comment (nolanlawson) [inline .testcaferc.json:11]:
```suggestion
      "module": "axe-core/axe.js"
```

Might help with debugging.

**Review by nolanlawson — APPROVED** [review]
:+1: when tests are passing (requires the other PRs I imagine). This is awesome.

There are also some lint issues but you should be able to fix that with `npx standard --fix`.

File: .testcaferc.json (hunk 1 above)
Comment (NickColley) [reply .testcaferc.json:11]: I wonder if the minified script runs quicker which might add up in the long run if we start using it more. But probably not.

Comment (nolanlawson) [reply .testcaferc.json:11]: That's fair. If we end up needing to debug something, we can switch to `axe.js`.

**Comment by NickColley** [comment]
Closing as unmaintained, much love Nolan

**Pull request closed without merging**